php-frontend/src/main/java/org/sonar/php/parser/PHPGrammar.java

```diff
@@ -358,6 +358,7 @@ public ClassPropertyDeclarationTree CLASS_CONSTANT_DECLARATION() {
       f.classConstantDeclaration(
         b.zeroOrMore(ATTRIBUTE_GROUP()),
         b.optional(VISIBILITY_MODIFIER()),
+        b.optional(b.token(PHPKeyword.FINAL)),
```

Comment:
This does not allow for example `final public const ` although it seems to be allowed. We might need to introduce something like `MEMBER_MODIFIER()` but for class constants.